src/main/scala/Garage/Main.scala

```diff
@@ -0,0 +1,61 @@
+package Garage
+
+import Garage.{Car, Customer, Garage}
+import org.mongodb.scala._
+import org.mongodb.scala.model.Filters._
+import org.mongodb.scala.model.Updates._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.{Failure, Success}
+
+object Main extends App {
+  // Get connection
+  val mongoClient: MongoClient = MongoClient("mongodb://localhost")
```

Comment:
I'd like to see you getting the client, database and collections from a separate class/object where you can get them based on input rather than hard code them